packages/experimental/RadioGroup/src/RadioGroup/useRadioGroup.ts

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { RadioGroupInfo, RadioGroupProps, RadioGroupState } from './RadioGroup.types';
+import { useSelectedKey as useValue } from '@fluentui-react-native/interactive-hooks';
+import { View } from 'react-native';
+
+export const useRadioGroup = (props: RadioGroupProps): RadioGroupInfo => {
+  const { value, defaultValue, onChange } = props;
+
+  // This hook updates the Selected Button and calls the customer's onClick function. This gets called after a button is pressed.
```

Comment:
Is comment about Button or RadioButton?

Reply:
It is for the selected Radio. Updated the wording to reflect that!